docker/scripts/entrypoint.sh

```diff
@@ -54,6 +54,7 @@ get_arg_name_from_env_name() {
     arg_name="${arg_name//_/-}"
     arg_name="${arg_name,,}"
     arg_name=--"${arg_name}"
+    arg_name=$(echo "$arg_name" | sed -r 's/^---+/--/')
```

Comment:
Please, can you explain here why you need this replacement? When `arg_name` already contains some dashes in front of it?

Reply:
With this sed command I'm not adding the double hyphens but I'm catching the cases in which we get 3 hyphens and replacing them with just two (or it will error). This happens because the script is not robust and, if the parameters value has an underscore more, it will append more hyphens than expected (this was happening on zkv-relay). 
The script will be more radically updated later one by completely removing the substitution logic to have something more robust.